client/consensus/aura/src/lib.rs

```diff
@@ -960,7 +984,7 @@ mod tests {
 		let res = executor::block_on(worker.on_slot(SlotInfo {
 			slot: 0.into(),
 			ends_at: Instant::now() + Duration::from_secs(100),
-			inherent_data: InherentData::new(),
+			create_inherent_data: Box::new(TestInherentDataProvider),
```

Comment:
```suggestion
			create_inherent_data: Box::new(()),
```

As you don't in the slots crate. Then you also can remove the test inherentdata provider